Intellifire climate (#23236)
home-assistant/home-assistant.github.io: source/_integrations/intellifire.markdown

To fully configure the IntelliFire integration you will need to enter your IntelliFire credentials which are the same ones you would use with the mobile app.

### Entities

The following controllable entities are available:

### Switch

- **Flame** - Turn fireplace on/off.
- **Pilot Light** - Turn pilot light on/off.

### Climate

> - **Thermostat** - If your unit is equipped with thermostatic control - this entry will be present. 

Reviewer: ```suggestion
- **Thermostat** - This entity will be present if your unit has thermostatic control.
```
